website/docs/components/modal/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,352 @@
+## Usage
+
+### When to use
+
+- When requesting an urgent or immediate response from the user.
+- As a confirmation when performing a destructive action or action that will impact other areas of the application.
+- For non-frequent tasks that can be interacted with quickly.
+
+### When not to use
+
+- When a user action is not immediately required.
+- In place of an alert, instead use a [Toast](/components/toast) or other alert type.
+- For complex editing, long forms, and experiences with a high level of detail. In these cases the task or function should exist at the page level.
+
+## Size
```

Comment:
at least in the alert example, these continue to be `###` most of the way through the content with `####` subheadings underneath them